Octokit/Models/Request/NewRelease.cs

```diff
@@ -32,7 +32,7 @@ public NewRelease(string tagName)
 
         /// <summary>
         /// Specifies the commitish value that determines where the Git tag is created from. Can be any branch or
-        /// commit SHA. Unused if the Git tag already exists. Default: the repository�s default branch
+        /// commit SHA. Unused if the Git tag already exists. Default: the repository’s default branch
```

Comment:
I think this is an encoding issue? Shouldn’t it be an apostrophe character?

Reply:
Reverted, this was an unintentional change.